packages/dcos-signal/extra/dcos-signal-ee.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=DC/OS Signal: reports cluster telemetry and analytics to help improve DC/OS
+
+[Service]
+Type=simple
+Restart=on-failure
+RestartSec=20
+LimitNOFILE=16384
+PermissionsStartOnly=True
+User=dcos_signal
+EnvironmentFile=/opt/mesosphere/environment
+EnvironmentFile=-/opt/mesosphere/etc/cfn_signal_metadata
```

Comment:
What does it mean if path is prefixed with `-`

Reply:
Citing the [docs](https://www.freedesktop.org/software/systemd/man/systemd.exec.html#EnvironmentFile=):

> The argument passed should be an absolute filename or wildcard expression, optionally prefixed with "-", which indicates that if the file does not exist, it will not be read and no error or warning message is logged.